scripts/apps/search/views/item-sortbar.html

```diff
@@ -1,19 +1,22 @@
 <div class="sortbar" ng-show="canSort()">
     <span ng-show="total" translate>Total:</span>
     <span ng-show="total" class="label-total">{{total}}</span>    
-    <div class="dropdown dropdown--align-right dropdown--hover" dropdown>
-        <button id="order_button" class="dropdown__toggle" dropdown__toggle>
+    <div class="dropdown dropdown--align-right" dropdown>
+        <button id="order_button" class="dropdown__toggle" dropdown__toggle aria-label="{{ 'Sort items' | translate }}">
```

Comment:
Shouldn't aria label match with text content inside the button?

Reply:
In this case, it's there to give more context to the action. The text in the button is accessible anyway, the aria-label is there as a description. I'll probably change the logic though and make the context visible to all users.